Structure README.md

sypets opened the pull request:
- reorder the part about creating issues and add a numbered list for better readability
- add subheaders in README.md
- link to CONTRIBUTING.md
- Add information about how to upvote issues

TPS commented:
@sypets Do you mind resolving the conflict I introduced by updating the GitHub contact link?

sypets commented:
Sorry about the late reaction. Is fixed now.

safinsingh commented on README.md:
```diff
@@ -9,22 +11,39 @@ Issues and feature requests posted to GitHub are not stored in any
 
-Post an issue if you have an issue or feature request for GitHub.
+
+## If you have an issue or feature request for GitHub.
```
Extra period in heading

safinsingh commented on README.md:
```diff
@@ -9,22 +11,39 @@ Issues and feature requests posted to GitHub are not stored in any
+
+## If you have an issue or feature request for GitHub.
+
+1. Search for existing [issues](https://github.com/isaacs/github/issues) here first
```
You can drop the `first` at the end, it's already the first item in an ordered list.

safinsingh commented on README.md:
```diff
@@ -9,22 +11,39 @@ Issues and feature requests posted to GitHub are not stored in any
+## If you have an issue or feature request for GitHub.
+
+1. Search for existing [issues](https://github.com/isaacs/github/issues) here first
+2. If you did not find any existing issue for your topic, post a new issue. 
```
Extra period in single-sentence list item

sypets commented:
I cleaned up my workspace and deleted the fork because I did not realize I still had a PR open. I did not see any other possibility than to create a new PR: #1852 

I resolved all comments and added the changes. Sorry.
